packages/flex-plugin-webpack/src/devServer/pluginServer.ts

```diff
@@ -17,6 +19,7 @@ export interface Plugin {
 interface StartServerPlugins {
   local: string[];
   remote: string[];
+  version?: string[];
```

Comment:
Rename `version` to `versioned` and remove the `?` - it should be required - if there are none, then it should be an empty array